adapters/InMobi/InMobiAdapter/GADMAdapterInMobiBannerAd.m

```diff
@@ -0,0 +1,194 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "GADMAdapterInMobiBannerAd.h"
+#import <InMobiSDK/IMSdk.h>
+#include <stdatomic.h>
+#import "GADInMobiExtras.h"
+#import "GADMAdapterInMobiConstants.h"
+#import "GADMAdapterInMobiInitializer.h"
+#import "GADMAdapterInMobiUtils.h"
+#import "GADMInMobiConsent.h"
+#import "GADMediationAdapterInMobi.h"
+
+static CGSize GADMAdapterInMobiSupportedAdSizeFromGADAdSize(GADAdSize gadAdSize) {
+    NSArray<NSValue *> *potentialSizeValues =
+    @[ @(GADAdSizeBanner), @(GADAdSizeMediumRectangle), @(GADAdSizeLeaderboard) ];
+    
+    GADAdSize closestSize = GADClosestValidSizeForAdSizes(gadAdSize, potentialSizeValues);
+    return CGSizeFromGADAdSize(closestSize);
+}
+
+@implementation GADMAdapterInMobiBannerAd {
+    
+    id<GADMediationBannerAdEventDelegate> _bannerAdEventDelegate;
+    
+    /// Ad Configuration for the banner ad to be rendered.
+    GADMediationBannerAdConfiguration *_bannerAdConfig;
+    
+    GADMediationBannerLoadCompletionHandler _bannerRenderCompletionHandler;
```

Comment:
Can you rename it to: `_adLoadCompletionHandler`?

Reply:
done